drivers/si70xx/include/si70xx_internals.h

```diff
@@ -58,10 +55,15 @@ extern "C" {
  * @name Si70xx register values.
  * @{
  */
-#define SI70XX_ID_SI7006            (0x06)
-#define SI70XX_ID_SI7013            (0x0D)
-#define SI70XX_ID_SI7020            (0x14)
-#define SI70XX_ID_SI7021            (0x15)
+#if defined(MODULE_SI7006)
+#define SI70XX_ID                   (0x06)
+#elif defined(MODULE_SI7013)
+#define SI70XX_ID                   (0x0D)
+#elif defined(MODULE_SI7020)
+#define SI70XX_ID                   (0x14)
+#else /* MODULE__SI7021 */
```

Comment:
please don't forget the error case. someone might just depend on "si70xx", thus ommitting the define. A nice error message is helpful in that case.